README.rst

```diff
@@ -47,7 +47,7 @@
   :target: https://scikit-learn.org/
 
 **scikit-learn** is a Python module for machine learning built on top of
-SciPy and is distributed under the 3-Clause BSD license.
+SciPy is distributed under the 3-Clause BSD license.
```

Comment:
Current this line continues the sentence above, which means the "and" is required. The "is distributed" is referring to scikit-learn.

Can you rever this change?